src/main/java/com/qubit/solution/fenixedu/integration/cgd/services/studentPhoto/CgdStudentPhotoClient.java

```diff
@@ -65,11 +64,10 @@ public boolean isMemberNumberLikeIdDocumentNumber() {
 
         private Optional<Person> getPersonFromDocumentId() {
             Collection<Person> persons = Person.findPersonByDocumentID(this.memberNumber);
-            if (persons.size() > 1) {
-                logger.warn("Too many persons match the member number sent from CGD when using document id number.");
+            if (persons.isEmpty() || persons.size() > 1) {
```

Comment:
Shouldn't the size() > 1 and isEmpty() be dealt with separately?

We will not be able to debug size() > 1 errors, since it will fail silently